index.js

```diff
@@ -1,9 +1,26 @@
 var sha3 = require("crypto-js/sha3");
 var schema_version = require("./package.json").version;
+var Ajv = require("ajv");
+var contractSchema = require("./spec/contract.spec.json");
 
 // TODO: This whole thing should have a json schema.
 
 var TruffleSchema = {
+  // Return a promise to validate a contract object
+  // - Resolves as validated `contractObj`
+  // - Rejects with list of errors from schema validator
+  validateContractObject: function(contractObj) {
+    return new Promise(function (resolve, reject) {
+      var ajv = new Ajv();
+      var validate = ajv.compile(contractSchema);
+      if (validate(contractObj)) {
+        resolve(contractObj);
+      } else {
+        reject(validate.errors);
+      }
+    });
+  },
```

Comment:
I almost wonder if this is too explicit (and I would counter my own argument by asking if it matters one way or another). Still, I like `validate()` over `validateContractObject()`, especially in the case where there's only one type of thing that ever gets validated. If there are two types of things, I would definitely call for the explicit function name. What's your take on simply naming it `validate()`?

Additionally, why is this wrapped inside a promise? It looks as though `Ajv`'s `validate` function is synchronous, meaning you can have it throw instead of promisified. I recommend only wrapping it in a promise if it's doing asynchronous work or if there's a high likelihood we'll add asynchronous code to the function in the future.